website/pages/en/billing.mdx

```diff
@@ -2,104 +2,84 @@
 title: Billing
 ---
 
-> Invoices are generated on a weekly basis.
+## Subgraph Billing Plans
 
-There are two options for paying for your query fees:
+There are two plans to use when querying subgraphs on The Graph Network. You can learn more about these plans on our Subgraph Pricing Page (coming soon).
```

Comment:
Not sure it makes sense to tease the Pricing page? Also are we calling it the Subgraph Pricing Page?

Reply:
Removed!